### PR TITLE
New traitor uplink item

### PR DESCRIPTION
**Pull request by s-h-d**
This PR adds a new traitor uplink item: The agent reveal
It costs 10 TC and gives you a lockbox containing a paper with the name of a random traitor (apart from the buyer). 
If you are the only traitor then the paper will just contain bullshit, and you wasted 10TC.
The lockbox can only be opened by traitors.
As to why you get a lockbox with a paper instead of just the paper, it's because I think it's cooler.

Also I made it 10 TC because I think it can be quite powerful, two traitors are stronger than the sum of their parts

 * rscadd: Added a new traitor uplink item, the "agent reveal"

### CODE REVIEW

**Comment by tuypo1** [comment]
Honestly I would make it just reveal all traitors but as is it's still pretty cool

**Comment by s-h-d** [comment]
Revealing all traitors seems too powerful I think, also it's pretty rare to have more than 3 traitors active at the same time anyway

**Comment by ShiftyRail** [comment]
Don't understand the point of this when code words exist

**Comment by boy2mantwicethefam** [comment]
I wouldn't mind but it's too expensive when you can say code words and traitors usually don't have a reason to dab on each other

**Comment by iFlashYou** [comment]
Can already see the traitor officer turning the rounds into extended

**Comment by DamianX** [comment]
Part of the reason why double agents was megagay was because of the following scenario:
- Traitor 1 turns himself in
- Unlocks his PDA and reveals John Doe is also a traitor
- Security collects the valid
- Traitor 1 completed his jecties and retires to sip cocktails in Hawaii

This was bad for many reasons but one of them was the _almost_ unquestionable validity of Traitor 1's claim.
Adding an item that provides a **totally** unquestionable statement about someone's antag status seems terrible because of this.

If your goal is to provide ways of increasing the likelihood of two traitors collaborating I would much rather see something that incentivizes the use of code words, like making them less obvious or something, or something that incentivizes the use of the syndicate encryption key, like maybe making traitors have the chance to start with one at no TC cost?

**TL;DR Collaboration good, antag test bad**

---

In addition to the paper being bad, its container is equally bad. It only opens for traitor minds which means it's just another antag test.

And even if the ideas were good, which they aren't, I don't think the code even works at all due to dynamic not using the syndicate faction or something like that. Did you test this at all?

I hereby invite you to close and delete this.

**Comment by ShiftyRail** [comment]
I can temperate what Damian said to remind you that there's a standing admin ruling that "X is a traitor and my target as a double agent" wasn't deemed proof enough to valid someone.

I think it's reasonable to expect this ruling to extend to other sources of metagay infos.

**Comment by s-h-d** [comment]
I will close this since most people think it's a bad idea. The intention was to increase traitor collaboration not to get them to hunt each other. As for the testing, I did it by manually making myself a traitor, I wasn't aware it's different from the one dynamic makes you
